queue/queue.go

```diff
@@ -339,6 +339,7 @@ type queueData struct {
 	result interface{}
 }
 
+//nolint
```

Comment:
Same as above: I removed it and don't see warnings.

Reply:
Fixed.